.travis.yml

```diff
@@ -19,3 +19,12 @@ before_install:
 script:
   - git describe --tags --dirty
   - make api c env ext cxx4opencl
+
+deploy:
+  provider: releases
+  api_key: "FIXME - ADD GITHUB OAUTH TOKEN"
```

Comment:
I think this needs to be setup before we can merge these changes.

Syntactically I have an example using this `api_key` here (I used `github_token` - I forget why):

https://github.com/bashbaug/OpenCL-Docs/blob/travis-ci/.travis.yml#L42

I think there was some setup needed on the GitHub or Travis side to make this work also though - is that what you were referring to in your PR description?

Reply:
Yes! I would prefer to avoid using my personal token,  as the token apply to all repositories the user has write access to, and since I have such permission to quite many open source projects I would prefer to avoid using my personal token.

In this tutorial, they recommend creating a machine user: https://docs.travis-ci.com/user/deployment/releases/. I could create one but I was thinking to check whether we already have one for the repositories hosted at Khronos.

Reply:
Thanks to @Khronoswebmaster for setting up the machine user. 

@bashbaug do you add the token as a secret in your repository setting? I think I prefer your approach, for my own testing I have added the encrypted token in Travis file directly: https://github.com/AnastasiaStulova/OpenCL-Docs/commit/0a78370e3ee3c10eb6e61d00f1c494378eb9e2b2. But it doesn't look that neat.

Reply:
Repo secrets seems to be only available for the Github Actions, so I have added this via Travis settings.